HeterogeneousCore/AlpakaTest/test/testAlpakaModules.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+function die { echo Failed $1: status $2 ; exit $2 ; }
+
+TEST_DIR=${LOCALTOP}/src/HeterogeneousCore/AlpakaTest/test
+
+if [ "x$#" != "x1" ]; then
```

Comment:
I think you don't need the `x` is you wrap the values in quotes `""`

Reply:
Thanks. I see I have accustomed to a workaround for "ancient shells".